### DISCUSSION TITLE
APP.Dao().FindAuthRecordByToken() baseTokenKey?

### DISCUSSION THREAD

**Discussion by OfflinePing** [pocketbase/pocketbase#4434] [Q&A]
Hey,
I wanted to ask what I gotta input as the baseTokenKey for using that function. I got the token after Login but I don't quite know what I should pass as 2nd argument

**Comment by ganigeorgiev** [comment]
The base token keys are the one from the app [Settings](https://pkg.go.dev/github.com/pocketbase/pocketbase@v0.21.3/models/settings#Settings).

For the record auth token, that would be: `app.Dao().FindAuthRecordByToken(token, app.Settings().RecordAuthToken.Secret)`
